docs/wallet/index.md

```diff
@@ -5,46 +5,42 @@ description: Overview of digital wallets available to send, spend, and earn Celo
 
-Wallets are tools that create accounts, manage keys, and help users transact on the Celo network.
+Celo is designed to work seamlessly with a range of wallets, each offering features to meet different user needs. 
 
-:::warning
-It's important to be careful when choosing a wallet because they manage your secret account keys. You should only use reputable wallets that are well-maintained by organizations/people that you trust. We added links to the source code so you can see when it's been last updated. 
-:::
+The [Celo Native Wallets](#celo-native-wallets) section provides an overview of wallets that are optimized for the Celo network. These wallets allow users to fully benefit from Celo’s native functionalities, such as phone number mapping and fee abstraction.
 
-The [Celo Native Wallets](#celo-native-wallets) section shows some popular wallets that were built specifically for the Celo network. They often include features that more general wallets do not, like paying for fees with ERC20 tokens, like cUSD. The [Celo Compatible Wallets](#celo-compatible-wallets) section has wallets that can work with Celo but were built for other networks (like Ethereum) or through company partnerships (like Pesabase).
+The [Celo Compatible Wallets](#celo-compatible-wallets) section provides an overview of commonly used wallets wallets that support the Celo network.
 
 ## Celo Native Wallets
 
 ### [Valora](https://valora.xyz/)
 
-Valora is a mobile wallet focused on making global peer-to-peer payments simple and accessible to anyone. It supports the Celo Identity Protocol which allows users to verify their phone number and send payments to their contacts.
+Valora is a mobile wallet designed to make global peer-to-peer payments by phone numbers simple and accessible for everyone. Valora supports both Celo and other Ethereum networks.
 
 - [valoraapp.com](https://valora.xyz/)
 
-- Platforms: iOS, Android
+- Platforms: [iOS]([url](https://apps.apple.com/us/app/valora-crypto-wallet/id1520414263)), [Android]([url](https://play.google.com/store/apps/details?id=co.clabs.valora&hl=en_US))
 - Maintainers: [Valora](https://valora.xyz/)
 - Ledger support: No
 - [Source Code](https://github.com/valora-inc/wallet)
 
-
 ### [MiniPay](https://www.opera.com/products/minipay)
 
-MiniPay is a non-custodial 2MB wallet that allows users to send and receive stablecoins with meager transaction fees—less than 1 cent. It was first launched in Africa to assist people in sending and receiving stablecoins using mobile numbers.
+MiniPay is a stablecoin-based non-custodial wallet now also available as a standalone dapp. It enables instant transactions using a phone number and provides easy access to dApps directly within the browser, all while consuming less than 2MB of data.
 
-- Countries: Ghana, Nigeria, Kenya, South Africa, Uganda
-- Platforms: Android
+- Platforms: [Android]([url](https://play.google.com/store/apps/details?id=com.opera.minipay&hl=en_US)), iOS (Coming Soon)
```

Comment:
also available in Opera Mini app in the following countries: Ghana, Nigeria, Kenya, South Africa, Uganda